### ISSUE TITLE
Superquadrics and polyhedral elements in DEM

### ISSUE THREAD

**Issue by rafaelbravopareja** [KratosMultiphysics/Kratos#3762]
Dear Kratos DEM developers:

What is your opinion about  the difficulty to implement superquadric and polyhedral particles in Kratos? Is it possible to include them in the next kratos releases?

Thank you in advance
Rafael

**Comment by philbucher** [comment]
@KratosMultiphysics/dem

**Comment by maceligueta** [comment]
Hi @rafaelbravopareja , this has been a long discussion in the @KratosMultiphysics/dem team.
Both superquadrics and polyhedra need considerable effort, and currently nobody in the team is working on it.
As a first step, we decided to go for the multi-sphere approach to model non-spherical paricles, as it allows any geometry and it still uses spheres.
The next step would probably be the polyhedra. This will need a whole set of implementations regarding the contact between face-face, edge-face, edge-edge, vertex-face, vertex-edge and so on. Once this is done, adapting our DEM general strategy to consider these forces would not be complex, because we already use rigid bodies as polyhedra (which can only feel contact against spheres currently).
Whether we add superquadrics or not would be decided after the polyhedra.
In summary, we are not dedicating time to this currently, so it will not be in the next release. However, we think it would be great to have those polyhedra working indeed.

**Comment by philbucher** [comment]
can we close this?

**Comment by rafaelbravopareja** [comment]
Yes.

Thanks
